assets/static/js/dataset-submission.js

```diff
@@ -157,10 +157,20 @@ $(function() {
         $("#largeFileUri").val($(this).val()).trigger("change");
     });
 
+    $("#funderList").on("keyup change", function() {
+        $(this).valid();
+    });
+
```

Comment:
Is this needed, it will validate all the time, not just when the Red X is there.

Reply:
Removed it